.bash_prompt

```diff
@@ -57,6 +57,9 @@ else
   sexy_bash_prompt_symbol_color="" # NORMAL
 fi
 
+# Define the default prompt terminator character '$'
+sexy_bash_prompt_symbol='$'
```

Comment:
While it would be nice to force users to specify `#` as the symbol for root users, that would create a backwards incompatible change. Can we add an `if` here to check for `$UID` as with before?